idd/Energy+.idd.in

```diff
@@ -22669,6 +22671,77 @@ OtherEquipment,
+       \type object-list
+       \object-list SurfaceNames
+  A3 , \field Schedule Name
+       \required-field
+       \type object-list
+       \object-list ScheduleNames
+       \note units in Schedule should be fraction applied to design level of other equipment, generally (0.0 - 1.0)
+  A4 , \field Evapotranspiration Calculation Method
+       \note Model selection for caclulating evapotranspiration of indoor greenery system.
+       \note This rate can also be actuated with user-defined calculations, see EMS application guide for actuator details.
+       \type choice
+       \key Penman-Monteith
+       \key Stanghellini
+       \default Penman-Monteith
+  A5 , \field Lighting Method
+       \note Three different methods are provided here (LED; Daylight; LED-Daylight)
+       \type choice
+       \key LED
+       \key Daylight
+       \key LED-Daylight
+  A6 , \field LED Intensity Schedule Name 
+       \type object-list
+       \object-list ScheduleNames
+  A7 , \field Daylighting Control Name
+       \note If daylighting is used in the selected lighting methods (Daylight or LED-Daylight), 
+       \note users should define an object of Daylighting:Control to obtain the daylighting illumance level 
+       \note and an object for Daylighing:ReferencePoint for the daylighting sensor location in the thermal zone. 
+       \note The name of the object of Daylighting:Controls should be specified in this field. 
+       \type alpha 
+  A8 , \field LED-Daylight Targeted Lighting Intensity Schedule Name
+       \note This field defines targeted LED intensity level for indoor living wall systems. 
+       \note The schedule values can be any positive number representing targeted photosynthetic photon flux density (PPFD). 
+       \note Based on the available daylighting, the required LED lighting level and power will be automatically adjusted to meet the targeted LED intensity level. 
+       \type object-list
+       \object-list ScheduleNames
+  N1 , \field Total Leaf Area
+       \note The value is the one-sided leaf area of an indoor living wall. 
+       \note Based on the users’ input, LAI is calculated as the ratio of the total leaf area and the partition wall area. 
+       \note Typical LAIs are 1.0 for grass and 3.0 for bushes and shrubs. The maximum LAI is 2.0 for the IndoorLivingWall module in EnergyPlus. 
+       \note If the calculated LAI is greater than 2.0, the maximum value of 2.0 is used for LAI in the simulation. 
+       \units m2
+       \type real
+       \ip-units ft2
+  N2 , \field LED Nominal Intensity 
+       \note The value represents photosynthetic photon flux density (PPFD) of LED grow light. 
+       \note PPFD is measured in micro-mole per m2 per second (umol_m2s) which establishes exactly how many photosynthetically active radiation (PAR) photons are landing on a specific area.
+       \units umol_m2s
+       \type real
+       \ip-units umol_m2s
```

Comment:
I noticed it because OpenStudio's parser choked on it.

I find this `units` confusing the way it's written, and inconsistent with the rest of units used throughout the IDD. We use stuff like `W/(m3/s)`  and `W/(gal/min)` which is much clearer. I suggest changing it.

Reply:
I see this has been fixed via https://github.com/NREL/EnergyPlus/commit/4f36bf8a2a7d8cdf44371f274ab6dae514541eaa already by @JasonGlazer (probably due to IDF Editor)

Reply:
> I noticed it because OpenStudio's parser choked on it.
> 
> I find this `units` confusing the way it's written, and inconsistent with the rest of units used throughout the IDD. We use stuff like `W/(m3/s)` and `W/(gal/min)` which is much clearer. I suggest changing it.

@jmarrec` umol_m2s` is the unit for photosynthetic photon flux density (PPFD). It represents how many PAR photons are landing on a specific area. It is used for lighting intensity in indoor growing spaces.

Reply:
@LipingWang we understand the intention of the units but we changed "umol_m2s" to "umol/m-s" to be consistent with the way other units are expressed in EnergyPlus IDD. I did this already as part of the IDF Editor changes #10414 which have already been merged.